procdiscovery/pkg/process/process.go

```diff
@@ -10,12 +10,24 @@ import (
 	"github.com/odigos-io/odigos/common/envOverwrite"
 )
 
+const NodeVersionConst = "NODE_VERSION"
+const PythonVersionConst = "PYTHON_VERSION"
+const JavaVersionConst = "JAVA_VERSION"
+
+// envDetailsSeparatorMap is a map of environment variables and their separators
+var envDetailsSeparatorMap = []string{NodeVersionConst, PythonVersionConst, JavaVersionConst}
```

Comment:
🤔 this is a slice not a map

Reply:
haha, yeah it was a map and I changed it to a slice, forgot to change the name

Reply:
Anyway, returning it to a map